### PR TITLE
Add admin accounts

### PR DESCRIPTION
**Pull request by timcheadle**
Fixes https://github.com/resolvetosavelives/redapp/issues/56

Use [Devise](https://github.com/plataformatec/devise) to add admin account creation, login, authorization in the admin UI

### CODE REVIEW

**Comment by timcheadle** [comment]
This still needs feature specs and to add login helpers for the other admin controller/feature specs

**Comment by timcheadle** [comment]
Ready for review!

File: config/initializers/devise.rb [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+  
+  # ==> Controller configuration
+  # Configure the parent class to the devise controllers.
+  # config.parent_controller = 'DeviseController'
+
+  # ==> Mailer Configuration
+  # Configure the e-mail address which will be shown in Devise::Mailer,
+  # note that it will be overwritten if you use your own mailer class
+  # with default "from" parameter.
+  config.mailer_sender = 'please-change-me-at-config-initializers-devise@example.com'
```
Comment (govindkrjoshi) [inline config/initializers/devise.rb]: We should to change this to `no-reply@simple.org`.

**Comment by govindkrjoshi** [comment]
How do we expect to create all Admins from the rails console? Or should we add a CRUD interface for that?

**Pull request merged**